src/services/fileServices.js

```diff
@@ -60,6 +61,7 @@ const writeFile = async (file, data) => {
 const updateFileMetadata = (file, memory) => {
   const date = new Date(memory.Date);
   fs.utimes(file, date, date, () => {});
+  return updateExifData(file, memory.Date);
 };
```

Comment:
Shouldn't this be changed to asynchronous and the `updateExifData` call be awaited?

Reply:
https://jakearchibald.com/2017/await-vs-return-vs-return-await/